### PR TITLE
Fix verification to use implementation

### PR DESCRIPTION
**Pull request by twcrone**
This should be merged AFTER the verification plugin is updated an published to 3.3

### CODE REVIEW

**Comment by twcrone** [comment]
Waiting on maven repo to get new verify plugin

**Review by tbradellis — APPROVED** [review]
lgtm

File: settings.gradle [hunk 1]
```diff
@@ -52,12 +52,6 @@ include 'functional_test:weave_test'
 include 'functional_test:weave_test_impl'
 
 // JDK 9+ module system tests
-if (JavaVersion.current().isJava9Compatible()) {
```
Comment (meiao) [inline settings.gradle:55]: These only caused failures locally, right?
I think we should leave these for GHA for now.

Comment (twcrone) [reply settings.gradle:55]: um...not failures on GHA yet WITH these changes

**Review by meiao — APPROVED** [review]

**Pull request merged**